quoridor-app/app/component/board.js

```diff
@@ -22,23 +23,31 @@ const generateKey = (type, position) => {
   return `${type}:${position.column}${position.row}`;
 };
 
-const Board = ({ squares, pawns }) => {
-  if (!squares) {
+const isSquareDisabled = (moves, position) => {
+  const exist = moves.filter(
```

Comment:
You can use [some](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/some) for that